src/algorithm/dynamics.hpp

```diff
@@ -106,7 +106,45 @@ namespace pinocchio
     
     return a;
   }
-  
+
+  ///
+  /// \brief Computes the inverse of the constraint matrix [[M JT], [J 0]].
+  /// The matrix is defined when we call forwardDynamics. This method makes use of
+  /// the matrix decompositions performed during the forwardDynamics and returns the inverse.
+  /// The jacobian should be the same that was provided to forwardDynamics.
+  /// Thus you should call forward Dynamics first.
+  /// \param[in] model The model structure of the rigid body system.
+  /// \param[in] data The data structure of the rigid body system.
+  /// \param[out] MJtJ_inv inverse of the MJtJ matrix.
+  ///
+  template<typename Scalar, int Options, template<typename,int> class JointCollectionTpl,
+           typename ConstraintMatrixType>
+  inline void getMJtJInverse(const ModelTpl<Scalar,Options,JointCollectionTpl> &
```

Comment:
I think this is the wrong naming. It is JtMJ and not MJtJ

Reply:
[M Jt]
[J 0]
Reading from left to right, top to bottom.

Reply:
Then, I would call it the KKTContactDynamicMatrix

Reply:
getKKTContactDynamicMatrixInverse seems too long. are you sure of this?

Reply:
But much more explicit than the current solution.
Maybe something like getContactDynamicMatrixInverse is a bit shorter and as explicit as the previous solution.

Reply:
Done.